ios-base/DroidKaigi 2020/Sponsor/Views/SponsorViewController.swift

```diff
@@ -0,0 +1,136 @@
+import ios_combined
+import RxCocoa
+import RxSwift
+import SafariServices
+import UIKit
+
+final class SponsorViewController: UIViewController {
+    private let disposeBag = DisposeBag()
+
+    @IBOutlet private var collectionView: UICollectionView! {
+        didSet {
+            collectionView.register(
+                UINib(nibName: "SponsorCell", bundle: nil),
+                forCellWithReuseIdentifier: SponsorCell.identifier
+            )
+            collectionView.register(
+                UINib(nibName: "SponsorCategoryHeaderView", bundle: nil),
+                forSupplementaryViewOfKind: UICollectionView.elementKindSectionHeader,
+                withReuseIdentifier: SponsorCategoryHeaderView.identifier
+            )
+            collectionView.register(
+                UINib(nibName: "SponsorDividerView", bundle: nil),
+                forSupplementaryViewOfKind: UICollectionView.elementKindSectionFooter,
+                withReuseIdentifier: SponsorDividerView.identifier
+            )
+        }
+    }
+
+    private lazy var loadingIndicatorView: UIActivityIndicatorView = {
+        let indicator = UIActivityIndicatorView(style: .gray)
+        return indicator
+    }()
+
+    private lazy var retryButton: UIButton = {
```

Comment:
In my opinion.
I think it is better to put "view" from "xib" than "code".
What do you think about put from `xib`, and change `isHidden = false` if occur error ?

Reply:
Exactry! I use xib elsewhere, It's strange that I use the code here...

I will fix it to use xib.